landscape.yml

```diff
@@ -600,11 +600,16 @@ landscape:
             twitter: 'https://twitter.com/Storage_OS'
             crunchbase: 'https://www.crunchbase.com/organization/storageos'
           - item:
-            name: Swift
-            homepage_url: 'https://www.swiftstack.com/product/openstack-swift'
-            repo_url: 'https://github.com/openstack/swift'
+            name: SwiftStack
+            homepage_url: 'https://www.swiftstack.com/product'
             logo: 'http://d.not.mn/OpenStack_Project_Swift_vertical.png'
             twitter: 'https://twitter.com/swiftstack?lang=en'
+          - item:
+            name: Swift
+            homepage_url: 'https://docs.openstack.org/swift/latest/'
+            repo_url: 'https://github.com/openstack/openstack'
```

Comment:
Should probably take the opportunity to change that URL to something other than openstack/openstack.

Reply:
Oh- whoops - copy-pasta issue from a different patch.